sympy/functions/elementary/complexes.py

```diff
@@ -979,6 +1165,26 @@ def _eval_evalf(self, prec):
 
 
 def unbranched_argument(arg):
+    '''
+    Returns periodic argument of arg with period as infinity.
+
+    Examples
+    ========
+
+    >>> from sympy import exp, exp_polar, unbranched_argument
+    >>> from sympy import I, pi
+    >>> unbranched_argument(exp(10*I*pi))
+    0
+    >>> unbranched_argument(exp(25*I*pi))
```

Comment:
I'd actually try to recommend using `exp_polar` here which shows more intuitive behavior for the users who doesn't know that exp automatically wraps around `2*pi*I`

Reply:
Have added two examples with exp_polar and one with exp.